library/src/test/kotlin/com/livefront/gsonkotlinadapter/KotlinReflectiveTypeAdapterFactoryTests.kt

```diff
@@ -196,7 +220,6 @@ class KotlinReflectiveTypeAdapterFactoryTests {
         // matches the pre-constructed data.
         val expectedData = stringDataObject()
         val actualData = gson.fromJson<StringData>(FROM_JSON_STRING_DATA_JSON)
-        println(actualData)
```

Comment:
😨

Reply:
Yea, I must have left this in here from a previous round of testing.